content/docs/integrations/jira-token/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Generating Jira Token
+lastUpdated: '2021-03-16T21:00:00.0Z'
+description: How to create Personal Access Token for Jira plugin.
+---
+
+## Introduction
+
+In order to use the Jira plugin, Roadie needs an API token to communicate with Jira APIs.
+
+
+These are set within backstage at the following url:
+
+```text
+https://<tenant-name>.roadie.so/secrets
+```
+
+This page describes how to create and set up the API token.
+
+## Steps
+
+1. Start by visiting the user profile settings page for your Jira user: `https://id.atlassian.com/manage-profile/security/api-tokens`
+   * It is a good practice to create an automation/bot account to Jira for API usage
+2. Click on the "Create API token" button.
+
+   ![Personal API Tokens screen in Atlassian with no tokens defined](./create-api-token.png)
+
+3. Give the token a label, so you can recognize it later, e.g. "Roadie Backstage"
+4. Jira will show you the API token once, make note of this
+
+   ![New created token modal window with a possibility to copy the token](./new-token.png)
+
+5. The API token needs to be prefixed with the Jira username and base64 encoded
```

Comment:
What is the JIRA username? If it is the email of the user that we used to create the API token then would it make more sense to say that explicitly?

Reply:
:+1: